### ISSUE TITLE
Evaluate work needed for Core a11y tool bump violations

### ISSUE THREAD

**Issue by thatblindgeye** [patternfly/patternfly#8258]
The [patternfly-a11y version bump PR](https://github.com/patternfly/patternfly/pull/8038) flagged a11y violations for various components. The work needed to complete this issue involves:

1. Identifying what is causing the violations
2. Opening up followup issues for each component outlining what work needs to be done for that component, and adding it to the overall [Resolve violations from a11y tool bump Jira ticket](https://redhat.atlassian.net/browse/PF-3831)

---

**Jira Issue:** [PF-3834](https://redhat.atlassian.net/browse/PF-3834)

**Comment by thatblindgeye** [comment]
Closed via https://github.com/patternfly/patternfly/issues/8492
